packages/@okta/i18n/src/properties/login.properties

```diff
@@ -1471,3 +1471,6 @@ idx.error.registration.unavailable = Registration is currently unavailable.
 idx.error.user.not.assigned.to.app = User is not assigned to this application.
 
 idx.error.code.user_not_assigned = You are not allowed to access this app. To request access, contact an admin.
+
+# Custom Identifiers
+custom.identifiers.userNotResolved = Can't identify your account with {0}. Enter your username.
```

Comment:
Do you have to implement this in a view? Or will it be picked up by the property key automaticlly?

Reply:
My understanding is it will be picked up by the key automatically, but will ask SIW folks for confirmation, thanks.

Reply:
@glenfannin-okta This key will be included in the response to `POST /idp/idx/identify`, and one remediation is the same endpoint (if the enduser continues the SIW should hit this endpoint again). I will look into the transformer and override, thank you!

Reply:
Looked at the response and it will automatically be handled since it includes the key and params in the message object.